{"name": "test_set_with_redis_error", "status": "passed", "description": "Test set met Redis error.", "attachments": [{"name": "log", "source": "d65b1247-0084-4185-9491-38d8f6bb9f9f-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "66b9d296-2f36-4976-9d44-a9e59ce5a384-attachment.txt", "type": "text/plain"}], "start": 1753819436183, "stop": 1753819436184, "uuid": "ef8a3386-b263-4584-9904-745d80a9123d", "historyId": "a8ed7c5085d40454f5f5d13183d2e4c7", "testCaseId": "a8ed7c5085d40454f5f5d13183d2e4c7", "fullName": "tests.unit.core.test_redis_cache_coverage.TestRedisCacheCoverage#test_set_with_redis_error", "labels": [{"name": "parentSuite", "value": "tests.unit.core"}, {"name": "suite", "value": "test_redis_cache_coverage"}, {"name": "subSuite", "value": "TestRedisCacheCoverage"}, {"name": "host", "value": "Yannick-Mac-Gillavry-werk.local"}, {"name": "thread", "value": "90219-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.unit.core.test_redis_cache_coverage"}], "titlePath": ["tests", "unit", "core", "test_redis_cache_coverage.py", "TestRedisCacheCoverage"]}